package.json

```diff
@@ -33,12 +33,13 @@
   "scripts": {
     "changelog": "./t changelog",
     "prepublishOnly": "yarn build",
-    "build": "rimraf dist/ && rollup -c rollup.config.js",
-    "test": "yarn build && mocha mod.spec.js && deno test.ts"
+    "build": "rimraf dist test/dist mod.d.ts && rollup -c rollup.config.js && mv dist/mod.d.ts .",
```

Comment:
Let's not use `mv` as this build step won't work on windows. Let's use `declarationDir` in `tsconfig.json` instead.

Reply:
`declarationDir` didn't work, it always seems to generate the d.ts files inside `dist/` lol; not sure what's the precedence in between `outDir` and `declarationDir`; programmatically passing the it might work but that will change the `const` to `var` again with rollup :sweat_smile:

Reply:
Tested it - `declarationDir` seems to only work as expected when `tsc` is triggered directly and doesn't work with rollup and using the plugin to programmatically change config would generate `var` for `const` like before.